tests/test_openclhpp.cpp

```diff
@@ -1684,6 +1684,60 @@ void testKernelSetExecInfo()
 #endif
 }
 
+
+cl_int clSetKernelExecInfo_setSVMPointers(cl_kernel kernel,
+                                cl_kernel_exec_info param_name,
+                                size_t param_value_size,
+                                const void *param_value, int cmock_num_calls)
+{
+    TEST_ASSERT_EQUAL_PTR(make_kernel(0), kernel);
+    TEST_ASSERT_EQUAL_HEX(CL_KERNEL_EXEC_INFO_SVM_PTRS, param_name);
+    TEST_ASSERT(param_value == nullptr || param_value_size >= sizeof(void *));
+
+    std::array<void *, 2> arr =*(std::array<void *, 2>*) param_value;
+    TEST_ASSERT_EQUAL_HEX(*(int *)arr[0], 0xaa);
+    TEST_ASSERT_EQUAL_HEX(*(int *)arr[1], 0xbb);
+
+    return CL_SUCCESS;
+}
+
+void testKernelSetSVMPointers()
+{
+#if CL_HPP_TARGET_OPENCL_VERSION >= 200
+    clSetKernelExecInfo_StubWithCallback(clSetKernelExecInfo_setSVMPointers);
+  
+    int a = 0xaa;
+    int b = 0xbb;
+    cl::vector<void *> vec = { &a, &b };
+    cl_int ret = kernelPool[0].setSVMPointers(vec);
+
+    cl_int expected = CL_SUCCESS;
+    TEST_ASSERT_EQUAL_HEX(expected, ret);
+#endif
+}
+cl_int clSetKernelExecInfo_EnableFineGrainedSystemSVM(cl_kernel kernel,
+                                 cl_kernel_exec_info param_name,
+                                 size_t param_value_size,
+                                 const void *param_value, int cmock_num_calls)
+{
+    TEST_ASSERT_EQUAL_PTR(make_kernel(0), kernel);
+    TEST_ASSERT_EQUAL_HEX(*(bool*)param_value, true);
+    TEST_ASSERT_EQUAL_HEX(CL_KERNEL_EXEC_INFO_SVM_FINE_GRAIN_SYSTEM,param_name);
+    TEST_ASSERT(param_value == nullptr || param_value_size >= sizeof(cl_int));
+
+    return CL_SUCCESS;
+}
+void testKernelEnableFineGrainedSystemSVM()
+{
+#if CL_HPP_TARGET_OPENCL_VERSION >= 200
+    clSetKernelExecInfo_StubWithCallback(clSetKernelExecInfo_EnableFineGrainedSystemSVM);
+    bool svmEnabled = false;
+    cl_int ret = kernelPool[0].enableFineGrainedSystemSVM(&svmEnabled);
```

Comment:
Something looks a little funny here...

1. This is passing `svmEnabled = false` but the test is checking for the value `true`?
2. I don't think we should be passing `&svmEnabled` here since the argument to `enableFineGrainedSystemSVM` is just a `bool`.